kafka-ui-react-app/src/serviceWorker.ts

```diff
@@ -41,6 +41,7 @@ function registerValidSW(swUrl: string, config?: Config) {
               // At this point, the updated precached content has been fetched,
               // but the previous service worker will still serve the older
               // content until all client tabs are closed.
+              // eslint-disable-next-line no-console
```

Comment:
@Guzel738 Can you disable `no-console` rule for the whole file instead commenting single lines?